src/dotnet/GeneratorAgent/src/Azure.Tools.GeneratorAgent/Azure.Tools.GeneratorAgent.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Why is this set to falses?

Reply:
Because we couldn't get the internals visibility working without assistance from our favorite EngSys friends.   😄

Reply:
I know how to do this, here's the note I wrote down:

To set up IVT, VS command prompt -> (repo root)/artifacts/bin/(project)/Debug/net8.0> Sn -Tp (project).dll 
That will give you the public key, which you can then copy into AssemblyInfo.cs

Reply:
Also I have no idea what the context is so apologies if this isn't helpful 😆  I just saw the comment and figured I would send the info just in case